pkg/users/users.scss

```diff
@@ -56,6 +56,7 @@
 
 .account-column-one {
   min-width: 270px;
+  margin-right: 8px;
```

Comment:
There is a way to avoid all the hacky CSS and also allow the interface to look "better" for other languages. But the layout will have to change. This way, although the font baseline is not aligned, the horizontal misalignment is not very visible. And the vertical alignment is not an issue. The horizontal misalignment only becomes very visible when the page is viewed on a smaller screen.

In English:

![Screenshot from 2023-03-17 17-24-14](https://user-images.githubusercontent.com/34256123/225962552-c6459a0d-840f-419c-8038-8a03f7f5c335.png)

In German:

![Screenshot from 2023-03-17 17-21-47](https://user-images.githubusercontent.com/34256123/225962645-76b07343-1eef-426c-b37b-a50e5bf42d10.png)

On a smaller screen: (the horizontal misalignment that was initially solved is now present but not very visible

![Screenshot from 2023-03-17 17-38-50](https://user-images.githubusercontent.com/34256123/225965975-39e6186b-abdd-46b2-a080-7174939cc6dd.png)

German: (with the same screen-size as above)

![Screenshot from 2023-03-17 17-41-10](https://user-images.githubusercontent.com/34256123/225966161-fa9ebe91-3f2f-41f2-a589-d8c49e3547dd.png)

But considering that we might eventually redesign this page, is it productive to continue adjusting the look?

Reply:
I don't agree with right aligning the half row content. Let's indeed not put too much effort on this page since is about to change soon, but my suggestion is very small adjustment, compared to what you proposed. I especially want to avoid the magic numbers from  ' margin-right: 8px; '

Reply:
Alright, I'll implement your suggestion and push the code